lib/index.js

```diff
@@ -2,8 +2,8 @@
 
 // Load modules
 
-const Dns = require('dns');
-
+let Dns = require('dns'); // eslint-disable-line prefer-const
```

Comment:
@skeggse - Do you prefer `rewire` or is it worth it to utilize `proxyquire` to prevent the ESLint override?